templates/ingress.yaml

```diff
@@ -372,9 +372,6 @@ metadata:
   {{- end }}
 spec:
 {{- include "coder.ingress.tls" . }}
-  backend:
-    serviceName: dashboard
```

Comment:
Shouldn't this become cemanager 8080

Reply:
This appears to be an unused property... but @coadler is on the blame so he'll let us know

Reply:
This is the backend it sends traffic to when it matches none of the rules. It should probably be changed to cemanager

Reply:
Gotcha. Will update.